README.md

```diff
@@ -359,35 +358,36 @@ def _process_output(data, context):
 
 You can also bring in external dependencies to help with your data processing. There are 2 ways to do this:
 1. If your model archive contains `code/requirements.txt`, the container will install the Python dependencies at runtime using `pip install -r`.
+2. If you invoke a multi-model endpoint, only the universal `requirements.txt` file's Python dependencies will be installed at runtime. Dependencies specified in any `requirements.txt` file present in a model archive will not be installed.
 2. If you are working in a network-isolation situation or if you don't want to install dependencies at runtime everytime your Endpoint starts or Batch Transform job runs, you may want to put pre-downloaded dependencies under `code/lib` directory in your model archive, the container will then add the modules to the Python path. Note that if both `code/lib` and `code/requirements.txt` are present in the model archive, the `requirements.txt` will be ignored.
 
 Your untarred model directory structure may look like this if you are using `requirements.txt`:
 
-        model1
+        /opt/ml/models/model1/model
```

Comment:
why needs extra subdir model?

Reply:
I think that is the directory structure which is expected when we create the endpoint. We might need to confirm with the hosting team regarding this directory structure.